Mathlib/Algebra/Group/Aut.lean

```diff
@@ -17,7 +17,7 @@ The definition of multiplication in the automorphism groups agrees with function
 multiplication in `Equiv.Perm`, and multiplication in `CategoryTheory.End`, but not with
 `CategoryTheory.comp`.
 
-This file is kept separate from `Data/Equiv/MulAdd` so that `GroupTheory.Perm` is free to use
+This file is kept separate from `Algebra/Group/Equiv` so that `GroupTheory.Perm` is free to use
```

Comment:
```suggestion
This file is kept separate from `Algebra/Group/Equiv.lean` so that `GroupTheory.Perm` is free to use
```

Reply:
Such a file doesn't exist, it is a folder, I will add a `/*` at the end to make it clear that we mean a folder.